libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -232,6 +237,8 @@ class AC_PrecLand
 
     TargetState                 _current_target_state;  // Current status of the landing target
 
+    bool                       _plnd_active;            // user can use aux function to turn Copter PrecLand on/off
```

Comment:
```suggestion
    bool                       _active;            // user can use aux function to turn Copter PrecLand on/off
```

I'm really not sure about using the word "active" to describe this.  It's a bit ambiguous as it might also mean that it is actively tracking a target.

enabled/disabled would be more typical.

Reply:
ENABLED was already used in the parameter to "enable" the library, ie PLND_ENABLED.  So to keep it independent of that, I used "active". Happy to word it differently though.

Reply:
Perhaps look at where that existing function is used and see if we can untie it from the parameter?